### PR TITLE
Ft basic login

### PR DESCRIPTION
**Pull request by thoth-ky**
-> BAsic LOgin

### CODE REVIEW

**Comment by chalchuck** [comment]
I Saw you installed and configured `simple_form`, why are you not using it in the forms?

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<% provide(:title, "Log in") %>
+<h1>Log in</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
```
Comment (chalchuck) [inline app/views/sessions/new.html.erb:6]: you have configured simple_form ..but you are using the default form builder, why?

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 module UsersHelper
+    # Returns the Gravatar for the given user.
+  def gravatar_for(user, size: 80)
```
Comment (chalchuck) [inline app/helpers/users_helper.rb]: Whatever you have done here still does not solve the Single responsibility issue I had earlier.
extract a method that returns the gravatar_url then pass the url to the image_tag

**Review by chalchuck — COMMENTED** [review]
Please review, fix and push.

**Comment by chalchuck** [comment]
@jmutuku95 you can merge!

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (thoth-ky) [reply app/views/sessions/new.html.erb:6]: I tried using simple_form but it wasn't working so I just suspended using it for now

**Pull request merged**